iscsi/osfamilymap.yaml

```diff
@@ -42,6 +42,7 @@ Debian:
         - linux-image-generic
```

Comment:
`linux-image-generic` is a virtual package which does not exist in Debian anymore. Also, virtual packages support [be deprecated in SaltStack's next version](https://github.com/saltstack/salt/pull/47263). 

Would  you consider removing it from the list in this PR?

Reply:
Hi @javierbertoli thanks for reviewing - yes I have removed `linux-image-generic` from the list.